homeassistant/components/philips_js/strings.json

```diff
@@ -30,7 +30,10 @@
     "step": {
       "init": {
         "data": {
-          "allow_notify": "Allow usage of data notification service."
+          "allow_notify": "Allow notification service"
+        },
+        "data_description": {
+          "allow_notify": "Allow the usage of data notification service on TV instead of periodic polling. This allow faster reaction to state changes on the TV, however, some TV's will stop responding when this activated due to firmware bugs."
```

Comment:
Change to:
"This allows faster"

and either say "when this is activated" or "with this activated".

Reply:
Adjustments here: https://github.com/home-assistant/core/pull/101783